### PR TITLE
[AutoMM] Improve API doc

### PR DESCRIPTION
**Pull request by zhiqiangdon**
*Issue #, if available:*

*Description of changes:*
Improved the API doc.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/src/autogluon/multimodal/predictor.py [hunk 1]
```diff
@@ -40,13 +40,16 @@
...
-
-    The prediction can be either classification or regression. The feature columns can contain
-    image paths, text, numerical, and categorical values.
-
+    AutoMM is designed to simplify the fine-tuning of foundation models
+    for downstream applications with just three lines of code.
+    AutoMM seamlessly integrates with popular model zoos such as
+    HuggingFace Transformers (https://github.com/huggingface/transformers),
+    TIMM (https://github.com/huggingface/pytorch-image-models),
+    and MMDetection (https://github.com/open-mmlab/mmdetection),
```
Comment (AnirudhDagar) [inline multimodal/src/autogluon/multimodal/predictor.py:48]: Should we make them hyperlinks instead? Seems to be too cluttered in the description [here](http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-3772/9a4dac4/api/autogluon.multimodal.MultiModalPredictor.html).

**Review by AnirudhDagar — APPROVED** [review]
LGTM, just left a minor suggestion.

File: multimodal/src/autogluon/multimodal/predictor.py (hunk 1 above)
Comment (zhiqiangdon) [reply multimodal/src/autogluon/multimodal/predictor.py:48]: Adding hyperlinks in docstrings seems not straightforward. Feel free to add them if you want.

Comment (AnirudhDagar) [reply multimodal/src/autogluon/multimodal/predictor.py:48]: Should have suggested the change in review. I opened a follow up PR for the same #3777.

**Pull request merged**